includes/admin/settings/display-settings.php

```diff
@@ -21,12 +21,81 @@ function edd_admin_header() {
 	if ( ! edd_is_admin_page( '', '', false ) ) {
 		return;
 	}
-
 	$numberNotifications = EDD()->notifications->countActiveNotifications();
+	$current_page        = ! empty( $_GET['page'] ) ? $_GET['page'] : '';
+	$is_single_view      = ! empty( $_GET['view'] ) ? true : false;
+
+	$page_title = '';
+	switch( $current_page ) {
+		case 'edd-settings':
+			$page_title = __( 'Settings', 'easy-digital-downloads' );
+			break;
+		case 'edd-reports':
+			$page_title = __( 'Reports', 'easy-digital-downloads' );
+			break;
+		case 'edd-payment-history':
+			$page_title = __( 'Orders', 'easy-digital-downloads' );
+			break;
+		case 'edd-discounts':
+			$page_title = __( 'Discounts', 'easy-digital-downloads' );
+			break;
+		case 'edd-customers':
+			$page_title = __( 'Customers', 'easy-digital-downloads' );
+			break;
+		case 'edd-tools':
+			$page_title = __( 'Tools', 'easy-digital-downloads' );
+			break;
+		default:
+			if ( ! empty( $_GET['page'] ) ) {
+				$page_title = ucfirst( str_replace( 'edd-', '', $current_page ) );
```

Comment:
Casting a noun to uppercase is problematic from a i18n perspective. Maybe the default should just be "Downloads".

Reply:
I'm updating this going forward, the point here is to just guess for now, and if we truly want i18n, we'll update all the extensions to load in their own using the filter below.

Reply:
We'll be able to go through other extensions, and use this filter to apply our 'breadcrumb' in a future update:
`	$page_title = apply_filters( 'edd_settings_page_title', $page_title, $current_page );`